app/grants/models.py

```diff
@@ -1007,6 +1007,10 @@ def update_tx_status(self):
             self.success = split_tx_status == 'success'
             self.split_tx_confirmed = True
 
+        should_have_split_tx_id = self.created_on > timezone.datetime(2020, 1, 9)
```

Comment:
I think we also need to check if the contribution is not part of a recurring subscription